docs/docs/guides/tutorials/Nextjs.mdx

```diff
@@ -0,0 +1,750 @@
+# Using Hanko for a Todo app with Next.js 13 and Prisma
+
+In this tutorial, you’ll learn how to build a Todo app with the Next.js 13 popular “App Router” structure and understand some of the most important changes that come with it. We will build a fully functional todo app, handling the creation, updating when completed and the option to delete the todos.
+
+## Let’s set it up 
+At Hanko, we understand that finding the right stack for the project is a big step. In this guide, we will bring Next.js as the main character of the project, we will test  the Client vs Server Components. For the style, we will use Tailwind CSS. We will use Hanko for the login and registration, user management and logout. Prisma will handle the storage.
+
+### Initiliaze the next app
+To create a new Next.js app, we can use the `create-next-app` or `create-next-app@latest` command-line tool followed by the name of your choice for the app. Open your terminal in Visual Studio Code and run the following command: 
+
+```
+ npx  create-next-app@latest todo-nextjs-hanko 
+```
+
+Then you’ll  be asked some prompts on what you will use for the app. The project configuration options should look something like this:
+
+![Pre-config Next.js prompts](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/8djxl7jl42xl90wpps1b.png)
```

Comment:
@Esther-Lita the images right now are coming from dev.to. It would be good to have them locally (applies for all images)